### DISCUSSION TITLE
[How To] Wrong token

### DISCUSSION THREAD

**Discussion by DemonAngeX2** [tmknight/docker-nordvpn#72] [Q&A]
In logs of the container, i have a WRONG TOKEN but when i generate a new one on NordVPN website and pasted it in the container, i start it and i have the same error .... ???
![image](https://github.com/tmknight/docker-nordvpn/assets/111583033/246f6a2e-2404-45e5-b353-856acd7309c5)

**Comment by tmknight** [comment]
I shudder to think this is a bug since token is the only way to authenticate.  I suggest you run the image as is (no compose, etc) with bash as entry, then step through start of daemon and then login to see where you've gone wrong.

**Comment by DemonAngeX2** [comment]
Hello,
Tank you for your answer. I just tried to create a new container in my unraid with your image and i put an environment variable for token (same token that my other container)
Connexion worked and is good.

Is there any folder config that i can map in my array to backup thet config of the container and restore it when i have a problem ?

To fix the bug i will delete my old container and image and do a new one.

Thank you a lot !
Have a good day :)

**Comment by tmknight** [comment]
The basics of docker is to put all your env in compose or env file or declare them via run.
